### PR TITLE
chore: Changelog for 3.7.0.0

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

* Compile changelog for 3.7.
* Add changelogs to plugins.
* Add missing breaking change to 3.7 breaking changes doc.

https://konghq.atlassian.net/browse/DOCU-3797

### Testing instructions

Preview link: https://deploy-preview-7411--kongdocs.netlify.app/gateway/changelog/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -9,6 +9,431 @@ Changelog for supported Kong Gateway versions.
...
+  This fixes support for using the respective provider's native SDK.
+ [#12903](https://github.com/Kong/kong/issues/12903)
+  * AI Proxy now has a [`preserve` option for `route_type`](/hub/kong-inc/ai-proxy/configuration/#config-route_type), 
+  where the requests and responses are passed directly to the upstream LLM. This enables compatibility with any
+  models and SDKs that may be used when calling the AI services.
+ [#12903](https://github.com/Kong/kong/issues/12903)
+  * Added support for [streaming event-by-event responses](/hub/kong-inc/ai-proxy/how-to/streaming/) back to the client on supported providers.
+ [#12792](https://github.com/Kong/kong/issues/12792)
+  * **Enterprise-only feature**: Added support for [Managed Identity authentication](/hub/kong-inc/ai-proxy/how-to/cloud-provider-authentication/) 
+  when using the Azure provider with AI Proxy.
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:122]: I don't think I added this one to the AI Proxy plugin changelog, should we add this to that plugin changelog as well?

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -9,6 +9,431 @@ Changelog for supported Kong Gateway versions.
...
+ [#12908](https://github.com/Kong/kong/issues/12908)
+* Bumped LuaRocks from 3.9.2 to 3.11.0
+ [#12662](https://github.com/Kong/kong/issues/12662)
+* Bumped `ngx_wasm_module` to `91d447ffd0e9bb08f11cc69d1aa9128ec36b4526`
+ [#12011](https://github.com/Kong/kong/issues/12011)
+* Bumped `V8` to 12.0.267.17
+ [#12704](https://github.com/Kong/kong/issues/12704)
+* Bumped `Wasmtime` to 19.0.0
+ [#12011](https://github.com/Kong/kong/issues/12011)
+* Improved the robustness of `lua-cjson` when handling unexpected input
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:420]: This one seems like a fix instead of a dependency?

File: app/gateway/changelog.md [hunk 3]
```diff
@@ -9,6 +9,431 @@ Changelog for supported Kong Gateway versions.
...
+ [#12786](https://github.com/Kong/kong/issues/12786)
+* Bumped `lua-resty-openssl` from 1.2.0 to 1.3.1
+ [#12665](https://github.com/Kong/kong/issues/12665)
+* Bumped `lua-resty-timer-ng` to 0.2.7
+ [#12756](https://github.com/Kong/kong/issues/12756)
+* Bumped PCRE from the legacy `libpcre` 8.45 to `libpcre2` 10.43
+ [#12366](https://github.com/Kong/kong/issues/12366)
+* Bumped `penlight` to 1.14.0
+ [#12862](https://github.com/Kong/kong/issues/12862)
+* Added the package `tzdata` to the DEB Docker image for convenient timezone setting
```
Comment (cloudjumpercat) [inline app/gateway/changelog.md:408]: This one seems like a fix instead of a dependency?

**Review by cloudjumpercat — APPROVED** [review]
Left a few small suggestions, otherwise, everything else looks good to me!

File: app/gateway/changelog.md (hunk 3 above)
Comment (lena-larionova) [reply app/gateway/changelog.md:408]: It's adding a new dependency named `tzdata`. We don't normally have too many changelog entries like this, so it does look out of place, but there's no real other place for it to go.

File: app/gateway/changelog.md (hunk 2 above)
Comment (lena-larionova) [reply app/gateway/changelog.md:420]: Looks like an operesty patch: https://github.com/Kong/kong/pull/12904/files. Another uncommon change to a dependency, but a dependency nonetheless.

File: app/_hub/kong-inc/openid-connect/_changelog.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 ## Changelog
 
+### {{site.base_gateway}} 3.7.x
+
+* Added support for [DPoP (Demonstrating Proof-of-Possession) token validation](/hub/kong-inc/openid-connect/demonstrating-proof-of-possession/).
```
Comment (lena-larionova) [inline app/_hub/kong-inc/openid-connect/_changelog.md:5]:
```suggestion
* Added support for [DPoP (Demonstrating Proof-of-Possession) token validation](/hub/kong-inc/openid-connect/how-to/demonstrating-proof-of-possession/). 
```

**Pull request merged**